scripts/protocgen.sh

```diff
@@ -24,6 +24,9 @@ for dir in $proto_dirs; do
     if [ -d "../cosmos" -a "$dir" != "./proto" ]; then
       cp -r ../cosmos $home/api
       rm -rf ../cosmos
+
+      # remove /api/cosmos/app/v2 folder since v2 not support pulsar
```

Comment:
We shouldn't be adding gogo generated code to api, this will cause even more problems than we already have.